README.md

```diff
@@ -60,8 +60,7 @@ Note that these examples rely on the setup code in the `/WORKSPACE` file in the
 rules_oci supports two different registry implementation for the temporary storage within actions spawned by bazel.
 
 1. By default we recommend using `zot` as it stores blobs on disk, however it doesn't support `Docker`-format images.
-2. `crane` is memory hungry as it stores blobs in memory, leading to high memory usage.
-   However it supports both `OCI` and `Docker` formats which is quite useful for using `Docker` images pulled from the registries such as DockerHub.
+2. `crane` is less tested alternative, however, it supports both `OCI` and `Docker` formats which is required to make images with `Docker` media types work.
```

Comment:
what do you mean by "less tested"? We've certainly got way more miles on it than Zot.

Reply:
As in corner cases might not be handled well. It has little no test coverage in its repo which makes it brittle to breakage

Reply:
I should reword it